JS03-04.html

```diff
@@ -9,19 +9,19 @@
 <body>
   <h1>ToDoリスト</h1>
   <form class="flex-list" name="radioBtns">
-      <input type="radio" id="all" name="status">すべて
+      <input type="radio" id="all" name="status" checked="checked">すべて

```

Comment:
`checked`のみでチェックマークがはいります。
```suggestion
      <input type="radio" id="all" name="status" checked>すべて
```